### PR TITLE
'as an admin' was too vague

### PR DESCRIPTION
**Pull request by NCBrady**
'as an admin' doesn't help users trying to understand how this works, so i've added in the bit above

### CODE REVIEW

**Comment by traya1** [comment]
Learn Build status updates of commit _[48df13f](https://github.com/NCBrady/memdocs/commits/48df13fe9ae86c479cb89567bff9a4c44d7741bb)_: 

### :warning: Validation status: warnings


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[windows-365/business/device-management.md](https://github.com/NCBrady/memdocs/blob/patch-6/windows-365/business/device-management.md) | :warning:Warning | | [Details](#user-content-1e1573f5d05382b162f28d84cead3537-6fd8aeb1)

<a id="1e1573f5d05382b162f28d84cead3537-6fd8aeb1"></a>
### [windows-365/business/device-management.md](https://github.com/NCBrady/memdocs/blob/patch-6/windows-365/business/device-management.md)
 - **Line 35, Column 1**: **[Warning: code-block-indented - [See documentation](https://review.learn.microsoft.com/help/contribute/validation-ref/code-block-indented?branch=main)]** `````Indented code blocks aren't allowed. Use a Markdown code block surrounded by triple backticks (```).`````

For more details, please refer to the [build report](https://buildapi.docs.microsoft.com/Output/PullRequest/aa268e4d-aaaf-30c8-4118-35d62ca794c9/202212282026493939-3478/BuildReport?accessString=8a2ee41798f83c088f656233d3740c2e7e775548e643268703b3e4b378346e88).

If you see build warnings/errors with permission issues, it might be due to single sign-on (SSO) enabled on Microsoft's GitHub organizations. Please follow instructions [here](https://review.learn.microsoft.com/en-us/help/onboard/sso-troubleshooting?branch=main) to re-authorize your GitHub account to Learn Build.

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/aa268e4d-aaaf-30c8-4118-35d62ca794c9?tabName=brokenlinks).

**Note:** Your PR may contain errors or warnings or suggestions unrelated to the files you changed. This happens when external dependencies like GitHub alias, Microsoft alias, cross repo links are updated. Please use these [instructions](https://review.learn.microsoft.com/en-us/help/onboard/faq?branch=main&tabs=pr-validation-error-solution#q-why-do-i-get-pr-validation-result-with-cryptic-error-or-warning-messages-not-related-to-the-changes-i-make-in-my-working-branch) to resolve them.

For any questions, please:<ul><li>Try searching the learn.microsoft.com <a href="https://review.learn.microsoft.com/en-us/help/?branch=main">contributor guides</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Learn support channel</a></li></ul>

File: windows-365/business/device-management.md [hunk 1]
```diff
@@ -30,7 +30,12 @@ ms.collection: M365-identity-device-management
 
 # Manage Cloud PCs
 
-As an admin, you can manage Cloud PCs in your organization in various ways:
+You can remotely manage Windows 365 Business Cloud PCs by using the Microsoft 365 admin center or windows365.microsoft.com. Each supports several remote management actions. However, to use these remote actions, you must have either of the following Azure Active Directory (Azure AD) role-based access roles:
+
+    Global Administrator
+    Windows 365 Administrator
+
+As an user with one of the above admin roles assigned, you can manage Cloud PCs in your organization in various ways:
```
Comment (JohanFreelancer9) [inline windows-365/business/device-management.md:38]:
```suggestion
    - Global administrator
    - Windows 365 administrator

As a user with one of the above admin roles assigned, you can manage Cloud PCs in your organization in various ways:
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @NCBrady, kindly commit the suggested changes. Many thanks.

**Comment by NCBrady** [comment]
done

**Comment by traya1** [comment]
Learn Build status updates of commit _[5665dbb](https://github.com/NCBrady/memdocs/commits/5665dbbbb8d43967a5c45fcb5619bc8e48f3cfeb)_: 

### :warning: Validation status: warnings


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[windows-365/business/device-management.md](https://github.com/NCBrady/memdocs/blob/patch-6/windows-365/business/device-management.md) | :warning:Warning | | [Details](#user-content-1e1573f5d05382b162f28d84cead3537-89d8339e)

<a id="1e1573f5d05382b162f28d84cead3537-89d8339e"></a>
### [windows-365/business/device-management.md](https://github.com/NCBrady/memdocs/blob/patch-6/windows-365/business/device-management.md)
 - **Line 35, Column 1**: **[Warning: code-block-indented - [See documentation](https://review.learn.microsoft.com/help/contribute/validation-ref/code-block-indented?branch=main)]** `````Indented code blocks aren't allowed. Use a Markdown code block surrounded by triple backticks (```).`````

For more details, please refer to the [build report](https://buildapi.docs.microsoft.com/Output/PullRequest/aa268e4d-aaaf-30c8-4118-35d62ca794c9/202212290819186086-3478/BuildReport?accessString=53053d819e37b554d92bbf4232d8afdfd8fb8adaa09a435af8ab923d9d5d7d8a).

If you see build warnings/errors with permission issues, it might be due to single sign-on (SSO) enabled on Microsoft's GitHub organizations. Please follow instructions [here](https://review.learn.microsoft.com/en-us/help/onboard/sso-troubleshooting?branch=main) to re-authorize your GitHub account to Learn Build.

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/aa268e4d-aaaf-30c8-4118-35d62ca794c9?tabName=brokenlinks).

**Note:** Your PR may contain errors or warnings or suggestions unrelated to the files you changed. This happens when external dependencies like GitHub alias, Microsoft alias, cross repo links are updated. Please use these [instructions](https://review.learn.microsoft.com/en-us/help/onboard/faq?branch=main&tabs=pr-validation-error-solution#q-why-do-i-get-pr-validation-result-with-cryptic-error-or-warning-messages-not-related-to-the-changes-i-make-in-my-working-branch) to resolve them.

For any questions, please:<ul><li>Try searching the learn.microsoft.com <a href="https://review.learn.microsoft.com/en-us/help/?branch=main">contributor guides</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Learn support channel</a></li></ul>

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by ErikjeMS** [comment]
#sign-off

**Pull request merged**